src/domain/Tables/Table/services/style.ts

```diff
@@ -223,13 +224,32 @@ const StyledDataCellChangeAnimation = keyframes`
   }
 `
 
-const StyledDataCell = styled.td`
+const StyledDataCell = styled.td<IStyledDataCellProps>`
   padding:  ${Variables.Spacing.sSmall}px;
   text-align: left;
   animation-name: ${StyledDataCellChangeAnimation};
   animation-duration: 500ms;
   animation-timing-function: ease-in;
 
+  ${(props: IStyledDataCellProps) => props.clickable && css`
+    & > span {
+      cursor: pointer;
+    }
+
+    span[role='button'] {
+      border-bottom: 1px transparent;
```

Comment:
Is using `text-decoration` more in line with the intention here?

Reply:
most likely